frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/SensitiveTopicsTooltipModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { TextWithTooltipButton } from "../../../commonComponents/TextWithTooltipButton";
+import Modal from "../../../commonComponents/Modal";
+
+interface Props {
+  tooltipText?: string;
+  modalTitle?: string;
+  showSyphilis?: boolean;
+}
+
+export const SensitiveTopicsTooltipModal = ({
+  tooltipText = "Why SimpleReport asks about sensitive topics like this",
+  modalTitle = "Why we ask for gender of sexual partners and other sensitive topics",
+  showSyphilis = false,
+}: Props): React.ReactElement => {
+  const [modalIsOpen, setModalIsOpen] = React.useState(false);
+
+  const openModal = () => setModalIsOpen(true);
+  const closeModal = () => setModalIsOpen(false);
+
+  return (
+    <>
+      <TextWithTooltipButton text={tooltipText} onClick={openModal} />
+      <Modal
+        showModal={modalIsOpen}
+        contentLabel={modalTitle}
+        onClose={closeModal}
+        containerClassName={"margin-1em"}
+      >
+        <Modal.Header styleClassNames={"font-sans-lg line-height-sans-3"}>
+          {modalTitle}
+        </Modal.Header>
+        <h3 className={"margin-bottom-05em"}>Gender of sexual partners</h3>
```

Comment:
I think the `<h3>`s here be `<h2>`s so we are not skipping heading levels.
<img width="516" alt="Screenshot 2024-06-05 at 18 35 22" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/15995651-308f-48fc-9e8d-4408a28fe6b6">

Reply:
Good catch!